### ISSUE TITLE
Fix or Remove Custom Java Scripting

### ISSUE THREAD

**Issue by joshuaskelly** [Interrupt/delverengine#267]
# Summary
Java scripting is currently broken because it requires the `JavaCompiler` class which is a JDK feature in Java 8 and not available in the JRE.

### Ship tools.jar
I've tried adding the `tools.jar` from the JDK libs directory to the JRE libs directory and adding the additional vmarg `-Xbootclasspath/a:jre/lib/tools.jar` to ensure it can find the library. This works okay from the command line, but does not work from the binaries we ship.

### Update Bundled JRE
I've also tried Java 17 now that it has entered an LTS state. It appears that the `JavaCompiler` is now included with the JRE but how we patch the class path was broken.

**Comment by PythooonUser** [comment]
Can be closed?

**Comment by joshuaskelly** [comment]
No. This has not be addressed either way yet. I've temporarily disabled custom scripting because it currently does not work.
